packages/sdks/snippets/react-sdk-next-pages/src/pages/announcements/[...page].tsx

```diff
@@ -5,77 +5,49 @@
  */
 import {
   Content,
-  fetchEntries,
   fetchOneEntry,
-  isEditing,
-  isPreviewing,
+  getBuilderSearchParams,
   type BuilderContent,
 } from '@builder.io/sdk-react';
-import type { GetStaticPaths, GetStaticProps } from 'next';
-import Head from 'next/head';
-import { useRouter } from 'next/router';
+import type { GetServerSideProps } from 'next';
 
 const BUILDER_API_KEY = 'ee9f13b4981e489a9a1209887695ef2b';
-const model = 'announcement-bar';
+const MODEL_NAME = 'announcement-bar';
 
-export const getStaticProps: GetStaticProps = async ({ params }) => {
```

Comment:
why teh change to getServerSideProps? we should stick to getStaticProps

Reply:
As we discussed last night, I've attempted to remove getStaticPaths, but it's throwing an error that says, "_getStaticPaths is required for dynamic SSG pages."

So, as per your suggestion and [This stackoverflow question](https://stackoverflow.com/questions/65783199/error-getstaticpaths-is-required-for-dynamic-ssg-pages-and-is-missing-for-xxx), I will go ahead and do it so that we won't be generating static paths for the section model.